pcdsdevices/radiation.py

```diff
@@ -0,0 +1,23 @@
+"""
+Radiation Sesnsor Classes.
```

Comment:
```suggestion
Radiation Sensor Classes.
```

Reply:
I think this was fixed in a previous commit.

Reply:
Did you forget to push? Github doesn't see a commit that fixes the typo

Reply:
Oh I see, I was looking in the /doc its the doc string in radiation.py 😉